src/rhsm/certificate.py

```diff
@@ -562,7 +562,7 @@ def write(self, pem_path):
         f.write(self.content)
         self.path = pem_path
         f.close()
-        os.chmod(pem_path, 0600)
+        os.chmod(pem_path, 0o600)
```

Comment:
Is the 0o on purpose? Also, could the constants be used in place? https://docs.python.org/3/library/os.html#os.chmod